src/resource-observer.js

```diff
@@ -0,0 +1,95 @@
+/*
+Copyright 2021 Splunk Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import {
+  InstrumentationBase,
+} from '@opentelemetry/instrumentation';
+
+import { version } from '../package.json';
+import { hrTime, isUrlIgnored } from '@opentelemetry/core';
+import { addSpanNetworkEvents } from '@opentelemetry/web';
+import { HttpAttribute } from '@opentelemetry/semantic-conventions';
+
+const MODULE_NAME = 'splunk-resource-obs';
+const defaultAllowedInitiatorTypes = ['img', 'script']; //other, css, link
```

Comment:
yes, what about link?

Reply:
I think other entry types will be just noise right now, some third party stuff etc. Most of this is loaded during page load for most sites. People can turn them on if they know what they are doing.